### PR TITLE
Fix all eslint problems

### PR DESCRIPTION
**Pull request by samsamson33**
There are two errors I wasn't sure about fixing:

1. At `commands\config.js 491:3`, `case "notify":` falls though into `case "list":`. I wasn't sure if this was intended or not. 
2. At `commands\deletecomment.js 16:133`, `prefix` is used without it being defined. I wasn't sure if the prefix should be `require`d from `config.json`, or if the server-specific prefix should be fetched and used.

The reason I was unsure about (2) is because many commands use the prefix defined in `config.json`. Is that intended, or shouldn't it all be the server prefix?

### CODE REVIEW

**Comment by BrightnessTM** [comment]
1. `notify` falling through to `list` is not intended.
2. `deletecomment` should be using the prefix from `config.json` because if there is no `qServerDB` there will inherently be no configured server prefix - so we'll use the default one.

File: commands/eval.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const config = require("../config.json");
-const core = require("../coreFunctions.js");
```
Comment (BrightnessTM) [inline commands/eval.js:2]: `coreFunctions` is included in order for easily evaling with `core.whatever()` without having to first require core.

File: commands/eval.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ module.exports = {
 		hidden: false,
 		permissions: ["VIEW_CHANNEL", "SEND_MESSAGES"]
 	},
-	do: async (message, client, args, Discord) => {
```
Comment (BrightnessTM) [inline commands/eval.js:12]: `Discord` is included in order for easily evaling with `Discord.whatever()` without having to first require it

File: commands/eval.js (hunk 1 above)
Comment (samsamson33) [reply commands/eval.js:2]: Why not remove it now, and just add it when you need it?

File: commands/eval.js (hunk 2 above)
Comment (samsamson33) [reply commands/eval.js:12]: If you want to keep that, how about using a [`varsIgnorePattern`](https://eslint.org/docs/rules/no-unused-vars#varsignorepattern) for variables that start with an underscore? So in
```js
do: async (message, client, _args, _Discord) => {
    /* ... */
}
```
Those args would be ignored, and it's clear that they're available if you need them (just remove the underscore)

**Comment by samsamson33** [comment]
There, that allows access to those from `eval`

**Review by BrightnessTM — APPROVED** [review]

**Pull request merged**